CoreArrangementPlugins/src/au/gov/asd/tac/constellation/plugins/arrangements/grid/GridArranger.java

```diff
@@ -276,37 +274,39 @@ public int getTopLeftVxId() {
     }
 
     /**
-     * Given the choice of grid (one of the GRID_CHOICE_xxx constants), return a
-     * setting from number of rows and number of columns. Zero means
-     * unconstrained.
+     * Given the choice of grid (one of the GRID_CHOICE_xxx constants), return a setting from number of rows and number
+     * of columns. Zero means unconstrained.
      *
      * @param gc The kind of grid to do the arrangement with.
      *
      * @return A Dimension corresponding to the choice of grid.
      */
     private static Dimension getGridSize(final GridChoice gc) {
-        if (gc == null) {
+        if (null == gc) {
```

Comment:
Don't know if this was a sonar suggestion or not but I think I prefer the null on the right of an comparison